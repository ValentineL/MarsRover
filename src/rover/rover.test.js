import React from 'react';
import { shallow } from 'enzyme';
import Rover from './rover';

describe('Rover', () => {
    const props = {
        ghost: true
    }

    it('should render Rover component', () => {
        const componentWithProps = shallow(<Rover {...props} />);
        expect(componentWithProps).toMatchSnapshot();

        const componentWithoutProps = shallow(<Rover />);
        expect(componentWithoutProps).toMatchSnapshot();
    });
});