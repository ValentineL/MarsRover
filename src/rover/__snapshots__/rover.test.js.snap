// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rover should render Rover component 1`] = `
<span
  className="rover undefined ghost "
>
  ðŸ›¦
</span>
`;

exports[`Rover should render Rover component 2`] = `
<span
  className="rover undefined  "
>
  ðŸ›¦
</span>
`;
